example-ideal.txt

```diff
@@ -0,0 +1,16 @@
+// A Picobot program that can fill an empty room
+
```

Comment:
True - although the `currentState` is part of the external DSL implementation, isn't ideal supposed to be the ideal syntax you arrived at in your `idea 7`?